src/clj_kondo/impl/linters.clj

```diff
@@ -475,19 +475,47 @@
            (node->line filename node
                        finding-type msg)))))))
 
-(defn lint-unused-bindings!
+(defn lint-bindings!
   [ctx]
   (doseq [ns (namespace/list-namespaces ctx)]
     (let [ns-config (:config ns)
           ctx (if ns-config
                 (assoc ctx :config ns-config)
                 ctx)
           ctx (assoc ctx :lang (:lang ns))]
+      (when-not (identical? :off (-> ctx :config :linters :used-underscored-binding :level))
+        (doseq [binding (sequence (comp (distinct)
+                                        (remove :clj-kondo.impl/generated)
+                                        (filter #(str/starts-with? (str (:name %)) "_")))
+                                  (:used-bindings ns))]
+          (findings/reg-finding!
+           ctx
+           {:type :used-underscored-binding
+            :filename (:filename binding)
+            :message (str "Using binding marked as unused: " (:name binding))
+            :row (:row binding)
+            :col (:col binding)
+            :end-row (:end-row binding)
+            :end-col (:end-col binding)})))
       (when-not (identical? :off (-> ctx :config :linters :unused-binding :level))
         (let [bindings (:bindings ns)
               used-bindings (:used-bindings ns)
               diff (set/difference bindings used-bindings)
               defaults (:destructuring-defaults ns)]
+          (when-not (-> ctx :config :linters :unused-binding
+                        :exclude-incorrectly-marked-unused)
+            (doseq [binding (set (->> (filter (comp not :clj-kondo.impl/generated) used-bindings)
+                                      (filter #(str/starts-with? (str (:name %)) "_"))))
+                    :when (not= "_" (str (:name binding)))]
```

Comment:
This is still here

Reply:
weird, i thought i'd removed that a while ago.